src/libs/actions/PersonalDetails.js

```diff
@@ -331,7 +331,7 @@ function setAvatar(file) {
                 return;
             }
 
-            setPersonalDetails({avatar: response.s3url, avatarUploading: false});
+            setPersonalDetails({avatar: response.avatarURL, avatarUploading: false});
```

Comment:
Do you thinkkk this is a place we could also update `avatarHighResolution`? I guess my question is really: do we **have to** update both the high & low res avatar URLs here, and if not, "should we" to make things more clear?

Reply:
I think we probably should also update avatarHighResolution, it looks like the OptionsList needs it to display the most up-to-date avatars. It also just makes sense to do that since we're going to have both URLs anyways.

https://github.com/Expensify/App/blob/3d015423f360108205be4c81876ba4f4c68f34c1/src/libs/OptionsListUtils.js#L113-L127

https://github.com/Expensify/App/blob/3d015423f360108205be4c81876ba4f4c68f34c1/src/libs/ReportUtils.js#L340-L351

Reply:
I'm starting to get a bit confused because it looks like we follow two different naming conventions for the high/los-res avatar urls:

Here is one of the places we use `avatar` and `avatarHighResolution`
https://github.com/Expensify/Web-Expensify/blob/4fb3c863ddadc87b1387d6047146c10a9d4f4fc5/lib/AppInit.php#L55-L55

Whereas here we use `avatar` and `avatarThumbnail`
https://github.com/Expensify/App/blob/3d015423f360108205be4c81876ba4f4c68f34c1/src/libs/actions/PersonalDetails.js#L102-L103

I'm thinking we should probably just standardize the naming everywhere (at least the parts that are used by App) if we're planning on updating this.

Reply:
I guess the question is, what are the names we'd like to assign to the low and high resolution avatar URLs?

I personally think `avatar` for low-res and `avatarHighResolution` make more sense, since in App we use `avatar` most frequently (to display the smaller avatars in reports), and we use `avatarHighResolution` only when someone clicks on report details. Going to change to use those two for now and we can change again later if you disagree.

@Beamanator

Reply:
To add to the confusion, it appears policies also have a different naming convention for their avatarURLs. I'm guessing policies were implemented before we also started storing thumbnails, so everywhere we reference avatars returned with policies or policy summaries they're called `avatarURL` (which is basically `avatarHighResolution`):

https://github.com/Expensify/App/blob/3d015423f360108205be4c81876ba4f4c68f34c1/src/libs/actions/Policy.js#L94-L95

https://github.com/Expensify/App/blob/3d015423f360108205be4c81876ba4f4c68f34c1/src/pages/workspace/WorkspaceSettingsPage.js#L40-L41

Reply:
So to summarize, there are three ways we interact with avatar urls, I think we should definitely standardize naming across all three since they all touch `UserAPI::uploadAvatar` (and quite frankly it's really confusing to follow)

For PersonalDetail objects, `avatarHighResolution` is stored as `avatar` (this will ultimately be removed later on and added in the Web-E PR):
https://github.com/Expensify/App/blob/f3c831dc85259143a841faa9fcd4e020c7dd65cc/src/libs/actions/PersonalDetails.js#L314-L336. 
Also, we randomly store `avatarThumbnail` in PersonalDetails objects whenever we call `json_encode($personalDetailsObject)` in PHP:
https://github.com/Expensify/Web-Expensify/blob/4fb3c863ddadc87b1387d6047146c10a9d4f4fc5/lib/PersonalDetail.php#L184-L191

For Policies we store `avatarHighResolution` as `avatarURL`.
https://github.com/Expensify/App/blob/f3c831dc85259143a841faa9fcd4e020c7dd65cc/src/libs/actions/Policy.js#L409-L430
3.

Reply:
@Beamanator Actually going to wait on doing this since it's a relatively big change. Gonna ask in #engineering-chat.

Reply:
Dude good call on all of this needing to be updated & cleaned up!